deploy/central/tasks-chart/templates/task-ods-build-java.yml

```diff
@@ -0,0 +1,157 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  name: ods-build-java{{ .Values.taskSuffix }}
+spec:
+  description: |
+    Builds Java Gradle applications.
+
+    Note on build environment variables available:
+    - ODS_OUTPUT_DIR: this environment variable points to the folder
```

Comment:
I'd encourage some more Asciidoctor formatting, such as surrounding the env vars in backticks so that they are rendered as code snippets.